Extension/src/LanguageServer/extension.ts

```diff
@@ -83,6 +100,119 @@ export function activate(activationEventOccurred: boolean): void {
     }
 }
 
+/**
+ * Generate tasks to build the current file based on the user's detected compilers, the user's compilerPath setting, and the current file's extension.
+ */
+async function getBuildTasks(): Promise<vscode.Task[]> {
+    const activeEditor: vscode.TextEditor = vscode.window.activeTextEditor;
+    if (!activeEditor) {
+        return [];
+    }
+
+    // Don't offer tasks for header files.
+    const activeFileExt: string = path.extname(activeEditor.document.fileName);
```

Comment:
This code is case-sensitive, right? It seems like it needs to handle both cases then.